eo-maven-plugin/src/main/java/org/eolang/maven/OyRemote.java

```diff
@@ -42,25 +45,16 @@ public final class OyRemote implements Objectionary {
     private final String template;
 
     /**
-     * Ctor.
+     * Constructor.
+     * @param tag Tag
+     * @throws IOException if fails
+     * @throws IllegalArgumentException if fails
      */
-    public OyRemote() {
-        this("master");
-    }
-
-    /**
-     * Ctor.
-     * @param hash The GitHub hash
-     * @todo #490:30m Resolve abbreviated hash to a proper hash.
-     *  In order to avoid collisions resolve hash
-     *  (or branch) to a complete sha-256 hash of the commit.
-     *  Use only sha-256 hashes as a caching criteria.
-     */
-    public OyRemote(final String hash) {
+    public OyRemote(final String tag) throws IOException, IllegalArgumentException {
```

Comment:
@yasamprom no need to mention unchecked exceptions in `throws` section. Your IDE should warn you about this, btw